index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Package Showcase</title>
+  <link rel="stylesheet" href="https://unpkg.com/open-props">
+  <style>
+    body {
+      font-family: var(--font-sans);
+      background-color: var(--gray-9); /* Dark background for body */
+      color: var(--gray-1); /* Light text color */
+      margin: 0 auto;
+      padding: var(--size-3);
+      max-width: var(--size-sm);
+    }
+
+    header {
+      color: var(--gray-3); /* Lightest text color */
+      padding-bottom: var(--size-2);
+      text-align: center;
+      font-size: var(--font-size-1); /* Larger text for the header */
+      margin: var(--size-2) auto;
+    }
+
+    ul {
+      list-style-type: none;
+      padding: 0;
+      margin: var(--size-2) auto;
+      background-color: var(--gray-8); /* Slightly lighter dark background for contrast */
+      box-shadow: var(--shadow-3); /* Prominent shadow for 3D effect */
+      border-radius: var(--radius-2);
+    }
+
+    li {
+      border-bottom: 1px solid var(--gray-7);
+      margin: 0;
+      transition: background-color 0.3s;
+    }
+
+    li:first-child {
+      border-radius: var(--radius-2) var(--radius-2) 0 0;
+    }
+
+    li:last-child {
+      border-radius: 0 0 var(--radius-2) var(--radius-2);
+    }
+
+    li:only-child {
+      border-radius: var(--radius-2);
+    }
+
+    li:hover {
+      background-color: var(--gray-6); /* Slightly lighter on hover for interactive feel */
+    }
+
+    a {
+      display: block;
+      padding: var(--size-3) var(--size-4);
+      text-decoration: none;
+      color: var(--gray-4); /* Subtle blue color for links */
+      transition: color 0.3s;
+      font-size: var(--font-size-2); /* Standard text size for links */
+    }
+
+    a:hover {
+      color: var(--gray-12); /* Slightly brighter blue on hover */
+    }
+  </style>
+</head>
+<body>
+<header>
+  <h1>Pillarbox Extensions</h1>
+</header>
+<ul>
+  <li><a href="packages/pillarbox-playlist/index.html">pillarbox-playlist</a></li>
+  <li><a href="packages/skip-button/index.html">skip-button</a></li>
+</ul>
```

Comment:
A general comment for possible future improvement. We could add a small description attached to the main title, as well as one for the plugins/components and maybe even a screenshot (?)

Reply:
We sure can!, I've created the issue here: #29